scripts/towncrier-draft-to-file.py

```diff
@@ -0,0 +1,14 @@
+from subprocess import check_call
+
+
+def main():
+    """
+    Platform agnostic wrapper script for towncrier.
+    Fixes the issue (#7251) where windows users are unable to natively run tox -e docs to build pytest docs.
+    """
+    with open("doc/en/_changelog_towncrier_draft.rst", "w+") as f:
```

Comment:
w+ is unnecessary, use w

if for no other reason than someone is going to scratch their head and have to look up what w+ does upon reading